include/spdlog/details/os-inl.h

```diff
@@ -355,7 +359,17 @@ SPDLOG_INLINE size_t _thread_id() SPDLOG_NOEXCEPT
     return static_cast<size_t>(::thr_self());
 #elif __APPLE__
     uint64_t tid;
-    pthread_threadid_np(nullptr, &tid);
+#    if (MAC_OS_X_VERSION_MAX_ALLOWED < 1060) || defined(__POWERPC__)
```

Comment:
Please add comment to explaini those ifdefs

Reply:
Could you please say if it is okay now?